gamemode/modules/chat/sv_chat.lua

```diff
@@ -0,0 +1,27 @@
+function RVR.canHear( listener, speaker )
```

Comment:
Can we put this in like `RVR.Chat`
You'd also need
```lua
RVR.Chat = RVR.Chat or {}
local Chat = RVR.Chat
```
To fit in with other modules